templates/cloud/thank-you.html

```diff
@@ -9,7 +9,7 @@
 {% block content %}
 {% include "shared/_thank_you.html" with thanks_context="enquiring about Cloud" %}
 
-	
+
 {% endblock content %}
 {% block footer_extra %}
 <!-- Google Code for Services Canonical &mdash; contact us Conversion Page -->
```

Comment:
Yea spotted that. Alot of moving code in this branch. Will do a polish branch after.